src/components/ItemViewSubheader/LinkedInstanceDetails.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import css from '../../View.css';
+
+const LinkedInstanceDetails = ({
+  item,
+  instance,
+}) => {
+  const boundWithCount = item?.boundWithTitles?.length;
+  const instancePublicationCount = instance.publication?.length;
+  const linkedInstanceTitle = (
+    <Link to={`/inventory/view/${instance.id}`}>
+      {` ${instance.title} `}
+    </Link>
```

Comment:
Is this correct or do we need sth like
```
<>{' '}<Link>{title}</Link>{' '}</>
```
so the whitespace surrounding the title is not part of the link? I don't know exactly how this will render.

Reply:
I removed this whitespace surrounding the title actually because the translation looks like this `"Instance: {title}."` so it renders correctly. Just copied this part of the code when refactored